setup.py

```diff
@@ -131,6 +133,13 @@
     #
     packages=find_packages(exclude=['contrib', 'docs', 'tests']),  # Required
 
+    # Specify which Python versions you support. In contrast to the 'Programming
+    # Language' classifiers above, 'pip install' will check this and refuse to
+    # install the project if the version does not match. If you support Python 2
+    # or have more complicated version requirements, see
+    # https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires
+    python_requires='>=3.5',
```

Comment:
This is at odds with the classifiers above. I'm also concerned about this being missed by folks that fork this repo to start their new project, and not understanding why their package isn't installable with Python 2. Let's make this:

```suggestion
    python_requires='>=2.7',
```

Reply:
You’re right, I copied over the suggestion from PR #53 and didn’t notice the inconsistency. To be consistent with the classifiers, however, Python versions 3.0 to 3.3 need to be excluded as [documented](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires).

Well, to be precise: With the new commit I made, `python_requires` allows Python 3.8, but the classifiers don’t. I think that’s fine they are used a bit differently (versions in `python_requires` are *and*-ed and classifiers are *or*-ed).